ServiceBased/README.md

```diff
@@ -0,0 +1,50 @@
+# Workflow Manager-Samples
+
+Sample tools that help administer workflow items.
+
+## Features
+These sample tools are script tools that utilize the Python API to perform various tasks against the ArcGIS Workflow Manager workflow items. They are broken out into subfolders by version.
+
+* [10_9_x](./10_9_x/) - These tools are compatible with ArcGIS Workflow Manager versions 10.9 and 10.9.1.
+
+   - The AdminTools.zip contains script tools that allow workflow admins to import, export, and transfer workflows.
+   - The RecPost_Tool.zip contains a script tool that may be published as a geoprocessing service to reconcile and post branch versions.
+
+* [11_x](./11_x/) - These tools are compatible with ArcGIS Workflow Manager versions 11.0 and later.
+
+
+## Instructions
+
+1. Ensure the requirements below are met on the machine where the admin tools will be ran.
+2. Download the admin toolbox and accompanying script files.
+3. Before running the admin tools, update the Config.py file with admin connection information for the workflow item and make sure it stays in the same location as the tool script files.
+
+## Requirements
+
+* ArcGIS API for Python 2.0.x
+* ArcGIS Pro 2.8 - 2.9
```

Comment:
Were you also going to update the Python version requirements for different versions of the tool?

Reply:
Good catch @llam001. I went in and made the change but I am going to leave off the 11.0 requirements and notes until I et tools into that folder get it in the samples.